### ISSUE TITLE
Can I add Some CheatSheet In your Repository

### ISSUE THREAD

**Issue by nihal-soni** [DiyaVj/Profile-Card-Collection#26]
This will be very beneficial for beginner

**Comment by DiyaVj** [comment]
Yes, you can but it will not be considered for hacktoberfest as such contributions are not valid there.

**Comment by nihal-soni** [comment]
No problem just want to contribute

**Comment by JASKIRAT11011** [comment]
@DiyaVj i want to contribute to this project

**Comment by DiyaVj** [comment]
@nihal-soni Your dedication to open source is appreciable✨. Please PR your contribution.

**Comment by DiyaVj** [comment]
@JASKIRAT11011 Feel free to contribute and follow [CONTRIBUTING.md](https://github.com/DiyaVj/Profile-Card-Collection/blob/main/CONTRIBUTING.md)

**Comment by nihal-soni** [comment]
I have a Created a pull request you haven't accepted yet.

**Comment by DiyaVj** [comment]
Pardon @nihal-soni ! Your PR is not visible to me.
